src/main/steps/enter-your-address/content.ts

```diff
@@ -27,8 +27,8 @@ export const form: FormContent = {
       label: l => l.postcode,
       labelSize: null,
       hideError: true,
-      validator: (value, form) => {
-        if (form?.myAddressIsInternational === Checkbox.Checked) {
+      validator: (value, form = {}) => {
```

Comment:
when does this method get called without the form argument?

Reply:
👍 Good shout, I've updated the types on this and other calls to it so I don't need to default it